src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -154,6 +154,7 @@ public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers,
                 client.SendFile(filename, preBuffer, postBuffer, TransmitFileOptions.UseDefaultWorkerThread);
 
                 client.LingerState = new LingerOption(true, LingerTime);
```

Comment:
>I don't understand, do you want me to make LingerTime longer to solve this problem?

What is the difference between using LingerTime and Shutdown?  Does Shutdown simply mean "unlimited amount of linger time"?